### PR TITLE
docs: add operating/use-cases.md

### PR DESCRIPTION
**Pull request by RayHuangCN**
issue: https://github.com/thanos-io/thanos/issues/3507
create a new page at docs/operating/use-cases.md and add the first case  about "Horizontal scaleable Prometheus with Thanos + Kvass"

### CODE REVIEW

File: docs/operating/use-cases.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+title: Use cases
+type: docs
+menu: operating
+---
+
+# Overview 
+
+There are many great use cases of  [Thanos](https://github.com/thanos-io/thanos), we collect them here for others to refer to.
```
Comment (bwplotka) [inline docs/operating/use-cases.md:9]:
```suggestion
There are many great use cases of [Thanos](thanos.io), we collect them here for others to refer to.
```

File: docs/operating/use-cases.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+## Horizontal scaleable Prometheus with Thanos + Kvass
```
Comment (bwplotka) [inline docs/operating/use-cases.md:11]:
```suggestion
## Horizontal scaleable Prometheus Scraping with Kvass
```

**Review by bwplotka — APPROVED** [review]
Thanks!

Minor suggestions, otherwise LGTM :muscle:

**Comment by bwplotka** [comment]
To fix last CI check, please run `make docs` locally and commit changes (:

**Comment by RayHuangCN** [comment]
Thanks for review! ( - : 
I had fixed last CI check.

**Comment by RayHuangCN** [comment]
@bwplotka I found that ```make docs``` always report error  on mac OS with built-in ```sed```, i solved it by installing ```gnu-sed```. I think his can be mentioned on ```CONTRIBUTING.md``` or somewhere.
May i create another PR to do this? ( - :

**Pull request merged**